docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,66 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from compatible files.
+
+Selecting Data Set
+==================
+
+.. seealso::
+
+    :ref:`Selecting Data Set<cubeviz-selecting-data>`
+        Documentation on selecting data sets in the jdaviz viewers.
+
+
+
+Adding New Viewers
+==================
+
+In the toolbar towards the top of the UI, there is a :guilabel:`picture with +` icon. By
+clicking this icon, you can add new viewers to the application. You can then select from the data
+that has been loaded into the application to be visualized in these extra viewers. You can then utilize some of the
+Imviz-specific features, like Linked Pan/Zoom.
+
+Defining Spatial Regions
+========================
+
+Spatial regions can be created the same way as in Cubeviz.
+
+.. seealso::
+
+    :ref:`Defining Spatial Regions<spatial-regions>`
+        Documentation on defining spatial regions in an image viewer.
+
+Blinking
+========
+
+Blinking is an Imviz-specific functionality that allows a user to quickly switch
+between viewing two images. This can be done by selecting the :guilabel:`hammer and screwdriver` icon
+and then clicking the :guilabel:`right arrow` icon. You can also access blinking
+by selecting the "B" key on your keyboard.
```

Comment:
Uppercase might make user think they need to do Shift+b when this is not the case.
```suggestion
by selecting the "b" key on your keyboard.
```